src/components/TextLink.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable indent */
+import React from "react";
+import styled from "styled-components";
+import { Link } from "gatsby";
+import PropTypes from "prop-types";
+
+import config from "../theme/config";
+const { color } = config;
+
+const StyledTextLink = styled(({ as }) => as)`
+  text-decoration: none;
+  color: ${color.accent};
+
```

Comment:
Can we add a semibold font-weight to the links, too? I think it would make them stand out a bit more against the page/paragraph.